tsconfig.json

```diff
@@ -13,11 +13,7 @@
     "resolveJsonModule": true,
     "esModuleInterop": true,
     "allowSyntheticDefaultImports": true,
-    "experimentalDecorators": true,
-    "typeRoots": [
-      "./server/@types",
-      "./node_modules/@types"
-    ]
```

Comment:
these dirs cause me all manner of chaos locally - server types doesnt exist anyway